src/planner/binder/tableref/bind_joinref.cpp

```diff
@@ -130,9 +130,20 @@ unique_ptr<BoundTableRef> Binder::Bind(JoinRef &ref) {
 	{
 		LateralBinder binder(left_binder, context);
 		result->right = right_binder.Bind(*ref.right);
-		result->correlated_columns = binder.ExtractCorrelatedColumns(right_binder);
-
-		result->lateral = binder.HasCorrelatedColumns();
+		bool is_lateral = false;
+		auto all_correlated_columns = vector<CorrelatedColumnInfo>();
```

Comment:
`all_correlated_columns` seems to be unused

Reply:
Thanks for pointing that out! We somehow missed it in the code cleanup.